pippo-core/src/main/java/ro/pippo/core/Application.java

```diff
@@ -248,7 +249,8 @@ public final RequestResponseFactory getRequestResponseFactory() {
      * @return
      */
     protected RequestResponseFactory createRequestResponseFactory() {
-        return new RequestResponseFactory(this);
+//        return new RequestResponseFactory(this);
+        return new GZipRequestResponseFactory(this);
```

Comment:
@decebals , instead of creating separate `RequestResponseFactory`(like `GZipRequestResponseFactory`), the base class itself could be used for this purpose right?
Because what if `Accept-Encoding` specifies `deflate` instead of `gzip`. Also, it would be easier to extend for other new features by using common `RequestResponseFactory`